utils/cachingStrategy.ts

```diff
@@ -62,12 +62,16 @@ export const flushIndexedDb = () => {
 
 export const clearSession = () => {
   try {
-    window.sessionStorage.clear()
-    window.localStorage.clear()
-    flushIndexedDb()
+    setTimeout(() => {
+      window.sessionStorage.clear()
+      window.localStorage.clear()
+      flushIndexedDb()
+    }, 100)
   } catch (error) {
     console.error(error)
   } finally {
-    window.location.reload()
+    setTimeout(() => {
+      window.location.reload()
+    }, 200)
```

Comment:
```
export const clearSession = () => {
  try {
      window.sessionStorage.clear()
      window.localStorage.clear()
      flushIndexedDb()
  } catch (error) {
    console.error(error)
  } finally {
    setTimeout(() => {
      window.location.reload()
    }, 100)
}
```
How about this one? Will it do the same thing

Reply:
Aright. Let it go

Reply:
![frozen_wide-a051d4ad0bc2b62f4976fc58d9a963450e63b359](https://user-images.githubusercontent.com/9987732/227556965-c71689be-5014-4617-b019-392cd0716f7f.jpg)